### ISSUE TITLE
Support for OpenStreetMaps

### ISSUE THREAD

**Issue by sarmong** [tommi1hirvonen/ExifNotes#48]
**Is your feature request related to a problem? Please describe.**
I am not using Google Services on my phone and thus cannot use the location feature of this great app.

**Describe the solution you'd like**
Completely move to OSM and not use Google Maps.

**Describe alternatives you've considered**
If Google Services are not found, fall back to OSM. And have a checkbox in settings if a user wants to switch to OSM having Google Services installed.

**Comment by tommi1hirvonen** [comment]
Related to #32

**Comment by sarmong** [comment]
Yes, however, my issue is coming from the place of not being able to use the location feature at all and not just desire to not have proprietary code there (although I think of it as beneficial too).

You mentioned that it will require a lot of effort to get rid of Google.
But what about just adding rudimentary OSM support in addition to Google? Pretty much just selecting a point on the map and retrieve the coordinates. 

I wish I could help, but I never worked with Kotlin. Though, I can try to investigate the OSM API and what is needed to make it work.

**Comment by DKMellow** [comment]
A workaround I am using on my degoogled phone is that I copy the coordinates of my current location from OrganicMaps and then when adding the location in ExifNotes I paste the coordinates in the search field and it can successfully find the and add the location to the metadata.

Hope this helps for anyone out there looking for a solution!
